backends/cuda/ceed-cuda-operator.c

```diff
@@ -277,9 +287,22 @@ static int CeedOperatorApply_Cuda(CeedOperator op, CeedVector invec,
       CeedChk(ierr);
     }
   }
+
   // Q function
-  ierr = CeedQFunctionApply(qf, numelements * Q, impl->qvecsin, impl->qvecsout);
-  CeedChk(ierr);
+  if (identityQF) {
```

Comment:
@YohannDudouit I added this as a naive implementation of the identity QFunction. This definitely can be improved upon.

Reply:
I'm not fan of polluting the qFunction code with this special case, especially because identity should really be handle at the operator level to be optimized (to avoid restriction/prolongation). Thus I think that having an identity qFunction should be enough for now:
```c++
__global__ void identity_apply(...) {
  const CeedScalar* u = fields.in[0];
  CeedScalar* v = fields.out[0];
  for(...){
   v[i] = u[i];
  }
}
```

Reply:
Note, this is an identity QFunction, not an identity operator. Restriction/prolongation are still needed.